doc/contributing/py-2-3-guide.rst

```diff
@@ -0,0 +1,117 @@
+=====================
+CKAN Python 2/3 Guide
+=====================
+
+This section documents the requirements for porting existing CKAN code for Python 3 support, and for creating new code that supports both Python 2 and Python 3.
+
+----------
+User story
+----------
+
+As a developer working on CKAN or CKAN Extensions, I want a clear guide, for writing code compatible with Python 2 and Python 3, so my code is consistent with common practice around CKAN, using a common set of tools.
+
+---------
+Use cases
+---------
+
+- Contributing Python 2/3 compatible code in CKAN Core
+- Contributing Python 2/3 compatible code in a CKAN Extension
+- Starting a new CKAN Extension with Python 2/3 compatibility
+
+--------
+Python 3
+--------
+
+From CKAN 2.9, CKAN will officially support Python 2 (≥ 2.6) and Python 3 (≥ 3.6).
```

Comment:
CKAN 2.9 will support Python 2 (≥ 2.7) and Python 3 (≥ 3.6).

CKAN 3.0 will only support Python 3 (≥ 3.6)